controller/api.php

```diff
@@ -498,19 +503,20 @@ public function match_bet(int $match_id): JsonResponse
                 throw new BadRequestError('team is not set');
             }
 
-            if (zone_util::bets()->has_bet($this->get_user_id(), $args['match_id'])) {
+            $bets = zone_util::bets();
+            $matches = zone_util::matches();
+
+            if ($bets->has_bet($this->get_user_id(), $args['match_id'])) {
                 throw new BadRequestError('already bet');
             }
 
-            if (zone_util::matches()->is_over($args['match_id'])) {
+            if ($matches->is_over($args['match_id'])) {
                 throw new BadRequestError('match is over');
             }
 
-            zone_util::bets()->place_bet(
-                $this->get_user_id(),
-                $args['match_id'],
-                (int) $data['team']
-            );
+            $team_ids = $matches->get_match_team_ids((int) $data['team']);
+            $team_id = $team_ids[(int) $data['team'] - 1];
+            $bets->place_bet($this->get_user_id(), $team_id);
```

Comment:
Das passt glaub ich nicht in unser aktuelles Tabellen-Schema. get_match_team_ids() gibt entweder 1 oder 2, in der bets Tabelle steht aber die tatsächliche Datenbank-Team-ID.

Reply:
hier hab ich nix an der logik veraendert. wenn da irgndwas nicht passt dann passt es auch jetzt schon nicht :P

Reply:
ah, ich seh den fehler :D in get_match_team_ids muss ich $args['match_id'] reingeben